modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/test/impl/CmmnJobTestHelper.java

```diff
@@ -43,8 +47,13 @@ public static void waitForJobExecutorToProcessAllJobs(final CmmnEngineConfigurat
 
             @Override
             public Boolean call() {
+                JobQuery jobQuery = cmmnEngineConfiguration.getCmmnManagementService().createJobQuery();
+                List<Job> jobs = jobQuery.list();
+                for (Job job : jobs) {
+                    System.out.println(job.getJobType() + " " + job.getCategory() + " " + job.getJobHandlerType() + " " + job.getScopeId() + " " + job.getId());
```

Comment:
Assuming as this is a work in progress that the `println` will become a log message or be removed.